### PR TITLE
Function changes

### PR DESCRIPTION
**Pull request by fokinv**
Changed the get_uuids() to get the uuids from the struct of the service when invoked, so we don't have to set the uuid vector manually in the tests.
Grouped the methods, the ones created with a macro are first, then the others.

### CODE REVIEW

**Comment by dati91** [comment]
CC @zakorgy

File: src/fake_device.rs [hunk 1]
```diff
@@ -174,6 +160,50 @@ impl FakeBluetoothDevice {
...
+
+    pub fn get_uuids(&self) -> Result<Vec<String>, Box<Error>> {
+        let cloned = self.gatt_services.clone();
+        let services = match cloned.lock() {
+            Ok(guard) => guard.deref().clone(),
+            Err(_) => return Err(Box::from("Could not get the value.")),
+        };
+        let gatt_services = services.into_iter().map(|s| s.get_uuid().unwrap_or("".to_owned())).collect();
+
+        let cloned = self.uuids.clone();
```
Comment (dati91) [inline src/fake_device.rs]: You should remove the `uuids` from the Device struct and remove the `set_uuids` as well.

**Review by dati91 — CHANGES_REQUESTED** [review]

**Comment by fokinv** [comment]
@dati91 @zakorgy r?

**Comment by zakorgy** [comment]
lgtm

**Review by dati91 — APPROVED** [review]

**Comment by dati91** [comment]
lgtm, but first squash the 2 uuid commits

**Pull request merged**